src/src/AdminBundle/Entity/Admin.php

```diff
@@ -13,8 +13,8 @@
  * @ORM\Entity(repositoryClass="AdminBundle\Repository\AdminRepository")
  * @UniqueEntity("email")
  */
-class Admin extends BaseEntity implements UserInterface {
-
+class Admin extends BaseEntity implements UserInterface, \Serializable {
```

Comment:
Len otazka, naco je ta Serializacia?

Reply:
Pri tomto som isiel podla navodu na spojazdnenie userov z databasy. Serialize tam bol popisany tu https://symfony.com/doc/current/security/entity_provider.html#understanding-serialize-and-how-a-user-is-saved-in-the-session

Je tam napisane, ze pravdepodobne to netreba, ale je to tam dobre mat

Reply:
Ok